query-graphs-js/src/graphPath.ts

```diff
@@ -159,6 +159,8 @@ type PathProps<TTrigger, RV extends Vertex = Vertex, TNullEdge extends null | ne
   /** If the last edge (the one getting to tail) was a DownCast, the runtime types before that edge. */
   readonly runtimeTypesBeforeTailIfLastIsCast?: readonly ObjectType[],
 
+  readonly lastIsInterfaceObjectFakeCastAfterNonCollecting: boolean,
```

Comment:
add a comment for this?

Reply:
Yeah. Writing such a comment, it occurred to me that there was no need to add a new field, and that I could just reuse the `subgraphEnteringEdge` field we already have to implement, so update slightly. It saves the field but it's also probably a bit clearer because more explicit.